### PR TITLE
chore(textures): remove unnecessary dependencies

### LINKED ISSUE #1785: textures module requires loaders.gl/core alpha version
Reported by @trxcllnt 

It is just a devDependency - can we remove it?

loader modules should not depend on core, but I seem to remember that there was a complication here..

### PR DESCRIPTION
**Pull request by morphoix**
Fixes #1785

### CODE REVIEW

**Review by ibgreen — APPROVED** [review]
Thanks for the quick PR. Note that per comments in the issue #1785  there are at least one more place where an alpha release is referenced.

**Comment by morphoix** [comment]
> Thanks for the quick PR. Note that per comments in the issue #1785 there are at least one more place where an alpha release is referenced.

ok, fixed

**Review by dryabinin-actionengine — APPROVED** [review]

File: modules/geotiff/package.json [hunk 1]
```diff
@@ -39,8 +39,5 @@
     "@loaders.gl/worker-utils": "3.0.0-alpha.18",
```
Comment (Pessimistress) [inline modules/geotiff/package.json:39]: Still not matching package version

File: modules/zarr/package.json [hunk 2]
```diff
@@ -32,8 +32,5 @@
   "dependencies": {
     "@loaders.gl/worker-utils": "3.0.0-alpha.18",
```
Comment (Pessimistress) [inline modules/zarr/package.json:33]: Ditto

**Pull request merged**